docs/user_guide/getting_started_with_moonstone.rst

```diff
@@ -141,5 +141,21 @@ Normalization
 Analysis
 """"""""
 
-Visualization
-"""""""""""""
+Plot
```

Comment:
It would be nice to have a `How does it work` part with a simple example of usage in Python.

Reply:
On it!